galasa-extensions-parent/dev.galasa.ras.couchdb/src/main/java/dev/galasa/ras/couchdb/internal/CouchdbRasStore.java

```diff
@@ -129,40 +119,24 @@ public CouchdbRasStore(IFramework framework, URI rasUri, HttpClientFactory httpF
 
 
     // Protected so that we can create artifact documents from elsewhere.
-    protected void createArtifactDocument() throws CouchdbRasException {
+    protected void createArtifactDocument() throws CouchdbException {
         Artifacts artifacts = new Artifacts();
         createArtifactDocument(artifacts);
     }
 
-    protected void createArtifactDocument(Artifacts artifacts) throws CouchdbRasException {
+    protected void createArtifactDocument(Artifacts artifacts) throws CouchdbException {
 
         artifacts.runId = this.runDocumentId;
         artifacts.runName = this.run.getName();
 
         String jsonArtifacts = gson.toJson(artifacts);
-
-        HttpPost request = requestFactory.getHttpPostRequest(this.rasUri + "/galasa_artifacts");
-        request.setEntity(new StringEntity(jsonArtifacts, StandardCharsets.UTF_8));
-
-        try (CloseableHttpResponse response = httpClient.execute(request)) {
-            StatusLine statusLine = response.getStatusLine();
-            if (statusLine.getStatusCode() != HttpStatus.SC_CREATED) {
-                throw new CouchdbRasException("Unable to store the artifacts document - " + statusLine.toString());
-            }
-
-            HttpEntity entity = response.getEntity();
-            PutPostResponse putPostResponse = gson.fromJson(EntityUtils.toString(entity), PutPostResponse.class);
-            if (putPostResponse.id == null || putPostResponse.rev == null) {
-                throw new CouchdbRasException("Unable to store the artifacts document - Invalid JSON response");
-            }
-
-            this.artifactDocumentId.add(putPostResponse.id);
-            this.artifactDocumentRev = putPostResponse.rev;
-        } catch (CouchdbRasException e) {
-            throw e;
-        } catch (Exception e) {
-            throw new CouchdbRasException("Unable to store the artifacts document", e);
+        PutPostResponse putPostResponse = createDocument("galasa_artifacts",jsonArtifacts);
```

Comment:
Perhaps use a constant for the `galasa_artifacts` database instead of hardcoding